src/ipinfoWrapper.ts

```diff
@@ -48,14 +48,37 @@ export default class IPinfoWrapper {
      * then LruCache is used as default.
      * @param timeout Timeout in milliseconds that controls the timeout of requests.
      * It defaults to 5000 i.e. 5 seconds. A timeout of 0 disables the timeout feature.
+     * @param countries An optional parameter representing custom countries data. 
```

Comment:
Need to update these docs as these are not the params anymore, but rather a part of the object

Reply:
ahhh, yes I forget that i'll update this as well